versions.json

```diff
@@ -2,5 +2,6 @@
 	"1.0.0": "0.9.20",
 	"1.0.1": "0.9.20",
 	"1.0.2": "0.9.20",
-	"1.0.3": "0.12.12"
+	"1.0.3": "0.12.12",
+	"1.0.4": "0.12.12"

```

Comment:
Version above was changed to 1.1.0 but have 1.0.4 here.

Reply:
Removed this change. Was thinking ahead.